aptos-node/src/lib.rs

```diff
@@ -648,8 +656,44 @@ pub fn setup_environment_and_start_node(
             mempool_client_receiver,
             peers_and_metadata,
         );
+    let (dkg_txn_pulled_tx, dkg_txn_pulled_rx) = aptos_channel::new(QueueStyle::FIFO, 1, None);
```

Comment:
nit: @zjma, these should probably be pulled into helper functions (i.e., follow the pattern of the other services above). Inlining everything here seems messy. Why not something like:
```
let (dkg_runtime, jwk_consensus_runtime) = services::start_dkg_and_jwk_consensus(...)
```

Reply:
It's by design actually: now vtxn_pool, dkg, jwk_consensus are all top-level components (just like consensus), so do the channels between them. This way it should be well aligned with the existing pattern? (DKG/JWK do have their own helper functions already)